driverconfig/config.yaml

```diff
@@ -108,18 +109,17 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.4.0
   - name: isilon
     configVersions:
-      - configVersion: v2.4.0
```

Comment:
Why is v2.4.0 removed? Please keep that as operator supports n-2 versions of the driver